### DISCUSSION TITLE
are there any Exceljs Tutorials, Examples or Documentation?

### DISCUSSION THREAD

**Discussion by richlysakowski** [exceljs/exceljs#1867] [Q&A]
This looks really interesting, but I don't see any Exceljs Tutorials, Examples or Documentation?   
Are there any full-blown examples for using Exceljs?   I am not a JavaScript developer (yet), and I would be willing to get my feet wetter on JavaScript with Exceljs.  

Are there any Python wrappers for Exceljs?  I know there are pure Python-based toolkits for Excel (xlsxwriter, etc.) that are very complete, but they are not for a web-only version of Excel.  If there's a way to use this completely with Python wrappers that would be easiest for me initially, but I am willing to try it out using Javascript directly.  

Thank you.

**Comment by Siemienik** [comment]
@richdevboston about wrappers, we have no wrappers for python. However, it is able to write a script in JS and run it under python: http://sweetme.at/2014/02/17/a-simple-approach-to-execute-a-node.js-script-from-python/

About learning resources, currently, it is only the README.md on the main page: https://github.com/exceljs/exceljs and I'm sure that the discussion is also a great place to get knowledge:)

Finally creating new questions is the best way to get comprehensive answers. I'm also available on Gitter: https://gitter.im/Siemienik/community
